README.md

```diff
@@ -28,19 +28,17 @@ As a pet sitter, you can provide the service of walks using your own schedule, p
 
 ## Installation
 
-**Pre-requisites**: 
-- installed Xcode with the version specified under the [Current development Environment](#current-dev-environment).
-
-1) clone the current repository by running the following command in shell:
-```bash
-$ git clone git@github.com:ios-course/ironfoudation-team-project.git
-```
-If you need more information, please check the [GitHub's documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
-
-2) open the file `Woof.xcodeproj` in Xcode.
-3) select your device as the destination.
-4) press the run button.
-If you don't know how to run the app on a real device, please check the [documentation](https://developer.apple.com/documentation/xcode/running-your-app-in-simulator-or-on-a-device).
+**Pre-requisites**: _installed Xcode with the version specified under the [current development environment](#current-dev-environment)._
```

Comment:
Where's the `current development environment` section?

Reply:
This section is at the beginning of this file. The link is points to it.
<img width="371" alt="Screenshot 2023-05-31 at 10 49 14 AM" src="https://github.com/ios-course/ironfoudation-team-project/assets/120586844/5103a824-9d49-413d-b617-dd48b42c04c6">

Reply:
Fixed the name.